includes/class-wc-tracker.php

```diff
@@ -111,9 +111,22 @@ private static function get_tracking_data() {
 		$data['inactive_plugins'] = $all_plugins['inactive_plugins'];
 
 		// Jetpack & WooCommerce Connect.
+		// Test if the site if a staging site based on the Jetpack criteria.
+		$is_jp_staging = false;
```

Comment:
Maybe this code to decide which method to use to determine if the site is a staging site depending on the Jetpack available should be abstracted to its own method?

Reply:
Agreed, was thinking about this today as well.